### PR TITLE
Migrating to DCO 1.1

### PR DESCRIPTION
**Pull request by mikeal**
We should use the latest version of important legal documents.

### CODE REVIEW

File: BasePolicies/Governance.md [hunk 1]
```diff
@@ -129,21 +129,31 @@ objections to adding or removing individual members, an effort must be
 made to resolve those objections. If consensus cannot be reached, a
 vote may be called following the process above.
 
-### Developer's Certificate of Origin 1.0
+# Developer's Certificate of Origin 1.1
```
Comment (rvagg) [inline BasePolicies/Governance.md]: needs to be `###`

**Comment by jasnell** [comment]
LGTM

**Comment by rvagg** [comment]
lgtm sans formatting nit

makes me wonder if there was some drama that caused the introduction of clause (d), some incident perhaps that caused a flare up somewhere because a company didn't realise contributions would be public and stick around forever

**Comment by thefourtheye** [comment]
LGTM

**Comment by williamkapke** [comment]
Whoops. My PR #95 did the same as this one. Sry to steal your thunder @mikeal ;)

**Pull request closed without merging**